README.md

```diff
@@ -202,6 +202,25 @@ storj.makePublic(bucketId, ["PULL", "PUSH"], function(e) {
 })
 ```
 
+### `storj.updateBucketById(bucketId, updateInfo, function cb(e) {})`
```

Comment:
I am not apposed to that all and like it better myself. The name currently follows the function name used in bridge-client